### DISCUSSION TITLE
Can a text field be set to multiline with enter putting in a line break?

### DISCUSSION THREAD

**Discussion by sheldonhull** [erikgeiser/promptkit#4] [Q&A]
Thank you for this really cool project to jump start using bubbletea components!
I love the tooling, but it's not easy to get started do to lack of friendly presets which you've offered here. Very cool!

I'm experimenting with having a multiline commit message option and need a text box that allows multiple lines. Is this something supported already or a feature request?

**Comment by erikgeiser** [comment]
I'm not currently planning on adding a multiline text input, I'm afraid. I'm kind of sure that I saw a multiline editor `bubbletea` model somewhere but I can't seem to find the repo right now.
